### PR TITLE
[IMP] subscriptions: update automation rules

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/web#id=3677982&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

**Comment by ksc-odoo** [comment]
@meng-odoo This one is ready for its first round of Peer Review, whenever you get a chance! 👍

File: content/applications/sales/subscriptions/automatic_alerts.rst [hunk 1]
```diff
@@ -1,83 +1,156 @@
...
+:guilabel:`Activity` section appears at the bottom of the :guilabel:`Automation Rules` form.
 
-This alert is applied to the *Rating Satisfaction* of your customers, and the action is triggered
-on *Timed condition*. If their satisfaction rate is lower than 50%, a salesperson contacts the
-customer. This action is automatically assigned to the salesperson who manages the subscription,
-and the due date is 5 days after the triggering of this action. This alert ensures that your
-clients are happy and that you are taking actions if they are not. It helps to keep your customer
-retention rates very high.
+- :guilabel:`Activity Type`: select an pre-configured activity type of drop-down menu.
+- :guilabel:`Title`: enter a custom title for chosen activity.
```
Comment (meng-odoo) [inline content/applications/sales/subscriptions/automatic_alerts.rst:144]:
```suggestion
- :guilabel:`Title`: enter a custom title for the chosen activity.
```

**Review by meng-odoo — APPROVED** [review]
Hi @ksc-odoo, great job on this doc, I only had small suggestions for wording and formatting. Then this will be good to go! Thanks :)

File: content/applications/sales/subscriptions/automatic_alerts.rst [hunk 2]
```diff
@@ -1,83 +1,156 @@
...
+   The number of subscriptions that match the configured criteria of the customized automation rule
+   are displayed at the bottom of the :guilabel:`Apply On` field.
+
+   If that green subscriptions link is clicked, Odoo reveals a separate page showcasing all the
+   subscriptions that meet that automation rule's criteria.
+
+Action section
+~~~~~~~~~~~~~~
+
+- :guilabel:`Action To Do`: choose the action that will occur once the automated rule is triggered.
```
[the remark this answers is not in the document]
Comment (ksc-odoo) [reply content/applications/sales/subscriptions/automatic_alerts.rst:86]: I encountered this, as well. I reformatted this lead-in portion to remedy that issue. Thanks for pointing it out! 👍

File: content/applications/sales/subscriptions/automatic_alerts.rst [hunk 3]
```diff
@@ -1,83 +1,156 @@
...
+   Do` field, the following field appears:
+
+   - :guilabel:`SMS Template`: create (and edit) a new SMS template *or* select from a list of
+     pre-configured SMS templates to send to the customer(s).
+
+   If :guilabel:`Set Contract Health value` is selected in the :guilabel:`Action To Do` field, the
+   following field appears:
+
+   - :guilabel:`Health`: designate the health of the subscription by choosing one of the following
+     options: :guilabel:`Neutral`, :guilabel:`Good`, or :guilabel:`Bad`.
```
[the remark this answers is not in the document]
Comment (ksc-odoo) [reply content/applications/sales/subscriptions/automatic_alerts.rst:114]: No longer a note. 👍

**Comment by ksc-odoo** [comment]
Thanks for the always-helpful feedback @meng-odoo -- I made all those suggested edits, and this one is now ready for a 2nd Round of Peer Review from the Materials team... 👍 @Felicious @hojo-odoo @brse-odoo

File: content/applications/sales/subscriptions/automatic_alerts.rst [hunk 4]
```diff
@@ -1,83 +1,159 @@
...
+
+The :guilabel:`Apply On` section dictates which subscription orders/customers this automated action
+applies to.
+
+- :guilabel:`MRR Between`: designate a range of monthly recurring revenue to target.
+- :guilabel:`MRR Change More`: designate a change of monthly recurring revenue to target, in either
+  percentage or unit of currency.
+- :guilabel:`Over`: choose a period of time over which the designated KPIs are calculated.
+- :guilabel:`Rating Satisfaction`: designate a :guilabel:`greater than` or :guilabel:`less than` a
+  percentage of satisfaction.
```
Comment (Felicious) [inline content/applications/sales/subscriptions/automatic_alerts.rst:59]:
```suggestion
- :guilabel:`Rating Satisfaction`: designate satisfaction as :guilabel:`greater than` or
  :guilabel:`less than` a percentage.
```

File: content/applications/sales/subscriptions/automatic_alerts.rst [hunk 5]
```diff
@@ -1,83 +1,159 @@
...
+Apply On section
+~~~~~~~~~~~~~~~~
+
+The :guilabel:`Apply On` section dictates which subscription orders/customers this automated action
+applies to.
+
+- :guilabel:`MRR Between`: designate a range of monthly recurring revenue to target.
+- :guilabel:`MRR Change More`: designate a change of monthly recurring revenue to target, in either
+  percentage or unit of currency.
+- :guilabel:`Over`: choose a period of time over which the designated KPIs are calculated.
```
Comment (Felicious) [inline content/applications/sales/subscriptions/automatic_alerts.rst:57]: possibly put KPI in an abbr tag and define what the acronym stands for?

File: content/applications/sales/subscriptions/automatic_alerts.rst [hunk 6]
```diff
@@ -1,83 +1,159 @@
...
-  :alt: Satisfaction survey in Odoo Subscriptions
+  :alt: A sample Automation Rules form in the Odoo Subscriptions application.
+
+Automation rule form fields
+---------------------------
+
+- :guilabel:`Action Name`: title of the automated action rule.
+
+Apply On section
+~~~~~~~~~~~~~~~~
```
Comment (Felicious) [inline content/applications/sales/subscriptions/automatic_alerts.rst:49]:
```suggestion
Apply on
~~~~~~~~
```
possibility for brevity and sentence case formatting. Feel free to reject this, as I know you have it written to match the section in the Automation Rule form (:

File: content/applications/sales/subscriptions/automatic_alerts.rst [hunk 7]
```diff
@@ -1,83 +1,159 @@
...
+
+.. warning::
+   When the :guilabel:`Trigger Now` button is clicked, Odoo will trigger the action on *all* linked
+   subscriptions, regardless of possible timed conditions.
+
+If :guilabel:`Timed Condition` is selected in the :guilabel:`Trigger On` field, the following fields
+appear:
+
+- :guilabel:`Trigger Date`: represents when the condition should be triggered. If left blank, the
+  action will be created at the creation of the subscription *and* when it is updated.
```
Comment (Felicious) [inline content/applications/sales/subscriptions/automatic_alerts.rst:134]:
```suggestion
  action is created upon subscription creation *and* updates.
```
optional

**Review by Felicious — APPROVED** [review]
This is a really well-written and detailed explanation of automatic features, @ksc-odoo ! 
It seems like such a complicated feature, but your doc makes it really easy to understand. Great job explaining all the fields in great detail :blush: 

I had some super minor suggestions, feel free to pass this onto to final review after this!

**Comment by ksc-odoo** [comment]
Thanks for the feedback, @Felicious ! I made all the suggested edits, and believe this one is now ready for Tech Review @samueljlieber 👍

File: content/applications/sales/subscriptions/automatic_alerts.rst [hunk 8]
```diff
@@ -1,83 +1,160 @@
...
+To view or modify any existing automation rule, simply click the desired rule from this page.
+
+.. note::
+   When modifying an existing automation rule, Odoo "grays-out" the :guilabel:`Action` section of
+   the form, and provides the following warning: *Action data can not be updated to avoid unexpected
+   behaviors. Create a new action instead.*
+
+To create a new automation rule, click :guilabel:`New`.
+
+.. image:: automatic_alerts/automation-rules-page.png
```
Comment (samueljlieber) [inline content/applications/sales/subscriptions/automatic_alerts.rst:32]: Indent the `:align:` and `:alt:` image attributes please :)
```suggestion
.. image:: automatic_alerts/automation-rules-page.png
   :align: center
   :alt: The Automation Rules page in the Odoo Subscriptions application.
```

File: content/applications/sales/subscriptions/automatic_alerts.rst [hunk 9]
```diff
@@ -1,83 +1,160 @@
...
-purchased that specific product. The survey will appear in the chatter of your respective
-subscription.
-
-.. image:: automatic_alerts/rating-satisfaction-survey.png
+  :alt: The Automation Rules page in the Odoo Subscriptions application.
+
+Clicking :guilabel:`New` reveals a blank :guilabel:`Automation Rules` form with numerous fields to
+configure.
+
+.. image:: automatic_alerts/automation-rules-form.png
```
Comment (samueljlieber) [inline content/applications/sales/subscriptions/automatic_alerts.rst:39]: Indent attributes here too please
```suggestion
.. image:: automatic_alerts/automation-rules-form.png
   :align: center
   :alt: A sample Automation Rules form in the Odoo Subscriptions application.
```

File: content/applications/sales/subscriptions/automatic_alerts.rst [hunk 10]
```diff
@@ -1,83 +1,160 @@
...
-  :alt: Satisfaction survey in Odoo Subscriptions
+  :alt: A sample Automation Rules form in the Odoo Subscriptions application.
+
+Automation rule form fields
+---------------------------
+
+- :guilabel:`Action Name`: title of the automated action rule.
+
+Apply on
+~~~~~~~~
```
Comment (samueljlieber) [inline content/applications/sales/subscriptions/automatic_alerts.rst:49]: Add "section" to the heading so it is consistent with the other headings
```suggestion
Apply On section
~~~~~~~~~~~~~~~~
```

File: content/applications/sales/subscriptions/automatic_alerts.rst [hunk 11]
```diff
@@ -1,83 +1,160 @@
...
+In the :guilabel:`Action To Do` field, choose the action that will occur once the automated rule is
+triggered. When clicked, the following options become available on a drop-down menu:
+
+  - :guilabel:`Create next activity`: creates the next activity to occur, which is configured in the
+    :guilabel:`Activity` section that appears at the bottom of the automation rule form.
+  - :guilabel:`Send an email to the customer`: sends an email to the customer(s) who fit the
+    specified criteria of the automation rule.
+  - :guilabel:`Send an SMS Text Message to the customer`: sends an SMS message to the customer(s)
+    who fit the specified criteria of the automation rule.
+  - :guilabel:`Set Contract Health value`: set the health value of the subscription contract.
```
Comment (samueljlieber) [inline content/applications/sales/subscriptions/automatic_alerts.rst:100]: No need for these bullets to be indented, please scoot them back :)
```suggestion
- :guilabel:`Create next activity`: creates the next activity to occur, which is configured in the
  :guilabel:`Activity` section that appears at the bottom of the automation rule form.
- :guilabel:`Send an email to the customer`: sends an email to the customer(s) who fit the specified
  criteria of the automation rule.
- :guilabel:`Send an SMS Text Message to the customer`: sends an SMS message to the customer(s) who
  fit the specified criteria of the automation rule.
- :guilabel:`Set Contract Health value`: set the health value of the subscription contract.
```

File: content/applications/sales/subscriptions/automatic_alerts.rst [hunk 12]
```diff
@@ -1,83 +1,160 @@
...
+field, the following field appears:
+
+  - :guilabel:`SMS Template`: create (and edit) a new SMS template *or* select from a list of
+    pre-configured SMS templates to send to the customer(s).
+
+If :guilabel:`Set Contract Health value` is selected in the :guilabel:`Action To Do` field, the
+following field appears:
+
+  - :guilabel:`Health`: designate the health of the subscription by choosing one of the following
+    options: :guilabel:`Neutral`, :guilabel:`Good`, or :guilabel:`Bad`.
```
Comment (samueljlieber) [inline content/applications/sales/subscriptions/automatic_alerts.rst:118]: No need for these bullets to be indented either
```suggestion
- :guilabel:`Email Template`: create (and edit) a new email template *or* select from a list of
  pre-configured email templates to send to the customer(s).

If :guilabel:`Send an SMS Text Message to the customer` is selected in the :guilabel:`Action To Do`
field, the following field appears:

- :guilabel:`SMS Template`: create (and edit) a new SMS template *or* select from a list of
  pre-configured SMS templates to send to the customer(s).

If :guilabel:`Set Contract Health value` is selected in the :guilabel:`Action To Do` field, the
following field appears:

- :guilabel:`Health`: designate the health of the subscription by choosing one of the following
  options: :guilabel:`Neutral`, :guilabel:`Good`, or :guilabel:`Bad`.
```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @ksc-odoo! Nice job on the improvements in this PR, the doc reads very well! I have a few corrections for you, please see below and tag me for another look when this PR is ready! 🙂

Also, this is a nitpick, the commit tag and PR title could be a little shorter/concise, something like:
```
[IMP] subscriptions: update automation rules doc
```

**Comment by ksc-odoo** [comment]
Alrighty @samueljlieber This one is ready for another look from ya, whenever you get a chance. Thanks! 👍

**Review by samueljlieber — APPROVED** [review]
Looks great @ksc-odoo, thank you! 

@StraubCreative this PR is good to go 👍

**Review by StraubCreative — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**